topics/imaging/tutorials/tutorial-CP/slides.html

```diff
@@ -0,0 +1,141 @@
+- Background detection: Detection of the foreground and subtraction from the image.
+---
+
+
+### 1) Segment nuclei
+
+  ![](../../images/tutorial-CP/subwf_nuclei.jpg)
+
+???
+Every CellProfiler pipeline needs to start by processing the metadata with the tool "Starting modules".
+
+- Segment complete nuclei: this tool identifies the nuclei within a certain size and excludes those cells that are touching the borders of the image.
+- Segmentation mask complete nuclei: gathers all the nuclei identified in the previous step in one mask.
+- Label and save labelled nuclei: adds the id of each nucleus on top of it. This can be useful later for a proper interpretation of the data analysis.---
+
+
+### Segmentation and labelling of the nuclei
+
+![identified_nuclei_with_labels](../../images/tutorial-CP/img_nuclei_labels.png "Identified nuclei with labels")
+
+???
+Resulting image after the first logical step of the workflow in which the nuclei were segmented and labelled.
+---
+
+
+### 2) Segment nucleoli
+
+![](../../images/tutorial-CP/subwf_nucleoli.jpg)
+
+
+???
+To identify the nucleoli the first step is to enhance the dark holes so that they can be segmented. 
+The segmentation of the dark holes will take place only when the dark holes fall inside a nucleus. This is to avoid the detection of wrong objects outside the cell.
+At this step, the segmentation of the nucleoli can be performed.
+It will be useful for visual exploration that the masks of the nuclei and nucleoli are saved together into one segmentation mask.
+---
+
+
+### Combined mask: nuclei + nucleoli
+
+![combined_mask_nuclei_nucleoli](../../images/tutorial-CP/img_combined_masks.png "Nuclei and nucleoli masks combined in which the nuclei are in blue and nucleoli in magenta.")
+
+???
+The combination of both masks gives an image with the nuclei in blue and the nucleoli in magenta.
+---
+
+
+### Background extraction
+
+![](../../images/tutorial-CP/subwf_background.jpg)
+
+???
+The extraction of the background can be useful to measure features that indicate that the images are artefacts.
+To get the background we first select the foreground and subtract it from the original image.
+We have partially detected the foreground in the nuclei segmentation in previous steps. However, that mask only included just those objects within a max and a min pixel size. We also discarded the images touching the borders. Here, we get rid of the constraints and segment all the nuclei.
+---
+
+
+### Feature extraction
+
+![](../../images/tutorial-CP/subwf_feature_extraction.jpg)
+
+- **Important:** Relate nucleoli and parent nucleus
+
+???
+The features can be extracted from the images and the objects. Depending on which type of object and the interest of the study, different parameters can be measured.
+The module that relates the objects is useful to export a table with the ids of the nucleoli in relation with their parent nuclei. That's useful for a meaningful interpretation of the data.
+---
```

Comment:
there is an empty slide before the key_points slide, if you remove this line it should be fixed :+1:

Reply:
fixed, thank you! :)